assets/scss/modules/_notifications.scss

```diff
@@ -106,15 +111,24 @@
     background-color: $shaded-bg-grey;
     border-color: $light-grey;
   }
+}
+
+.alert--borderless {
+  border: 0;
+  border-left-style: none;
+}
 
+.alert--short {
```

Comment:
not sure this is clear what it means, are we talking a squashed one? maybe `alert--slim`?